python/cuda_parallel/setup.py

```diff
@@ -112,6 +112,7 @@ def build_extension(self, ext):
     extras_require={
         "test": [
             "pytest",
+            "pytest-xdist",
```

Comment:
Note: be careful if we have large-size tests, because GPUs can easily go OOM if tests are running in parallel. In cuQuantum we had this enabled, but at QA time we found this often causes false positive on small GPUs like T4.

Reply:
Thanks for the warning. In automated testing (e.g. GitHub Actions) I'd add the `-n` option only when we really need it. In interactive testing it makes a big difference. I posted the timings in the PR description. For cuda_cooperative, `pytest -n 16` finishes after 27s, while it takes 322s with only one worker.